db/migrations/00005_create_storage_diffs_table.sql

```diff
@@ -2,14 +2,14 @@
 CREATE TABLE public.storage_diff
 (
     id             BIGSERIAL PRIMARY KEY,
+    address        BYTEA,
```

Comment:
Should this be an address_id?

Reply:
I think probably not - leaning against adding any additional work before writing raw diffs